packages/core/src/App/Components/Layout/Footer/go-to-deriv.jsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import { Icon, Popover, StaticUrl } from '@deriv/components';
 import { localize } from '@deriv/translations';
-import { deriv_urls } from '@deriv/shared';
 
 export const GoToDeriv = () => {
-    const message = localize(`Go to ${deriv_urls.DERIV_HOST_NAME}`);
+    const message = localize(`Go to Deriv.com`);
```

Comment:
` const message = localize('Go to {{hostname}}', {hostname: deriv_urls.DERIV_HOST_NAME});`